scripts/build.sh

```diff
@@ -47,9 +47,9 @@ echo "Start building Bytebase monolithic ${VERSION}..."
 echo ""
 echo "Step 1 - building bytebase frontend..."
 
-if command -v yarn &> /dev/null
+if command -v pnpm &> /dev/null
 then
-    yarn --cwd ./frontend && yarn --cwd ./frontend release
+    pnpm --cwd ./frontend && pnpm --cwd ./frontend release
```

Comment:
It should be `pnpm --cwd ./frontend i`.

---

BWT, I still prefer using `yarn` which is more popular then `pnpm` right now. So most newcomers will be a bit more familiar with the deploy process and contribution.

Reply:
The commands for package management are similar.
for pnpm most use command are,
```bash
pnpm i
pnpm up
pnpm add
pnpm remove
```
Not much learning cost，but pnpm saving disk space and boosting installation speed.
The better DX is worth to try.
by the way,pnpm have better monorepo support `pnpm workspaces` https://pnpm.io/workspaces. 
This may usefull in future.